src/security/context.cpp

```diff
@@ -255,7 +295,9 @@ class PolTaskCtx {
     try {
       ngx_log_debug(NGX_LOG_DEBUG_HTTP, req_log(), 0, "before task %p main",
                     this);
-      block_spec_ = static_cast<Self *>(this)->do_handle(*tp_log);
+      block_spec_ = as_self().do_handle(*tp_log);
+      // test long libddwaf call
+      // ::usleep(2000000);
```

Comment:
Stray comments?

Reply:
I just left marked where I should uncomment to test long libddwaf calls because it's not an easy thing to test otherwise.